githooks/pre-push

```diff
@@ -5,7 +5,7 @@
 
 
 echo Running ruff linter tests...
-TEST_RESULTS=$(ruff .)
+TEST_RESULTS=$(tox -e check-style)
```

Comment:
@amitschang FYI I requested changes based on my own code lol See https://github.com/ssec-jhu/base-template/pull/128#discussion_r1551735773

I think running it twice will be confusing when stuff fails but doesn't block.